core/embed/rust/src/ui/layout_eckhart/firmware/device_menu_screen.rs

```diff
@@ -1,4 +1,8 @@
+use core::ops::DerefMut;
+
 use crate::{
+    error::Error,
+    micropython::gc::GcBox,
```

Comment:
Please also add `#[cfg(feature = "micropython")]` to mod.rs for this module.

Reply:
IIUC, `ui::layout_eckhart::firmware` is marked with `#[cfg(feature = "micropython")]` in `core/embed/rust/src/ui/layout_eckhart/mod.rs`. 
Would it be enough?

Reply:
Ah right, sorry, thought this is in `layout_eckhart/component/`, never mind.